searx/engines/wordnik.py

```diff
@@ -0,0 +1,126 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""
+ Wordnik (general)
+"""
+
+import re
+from lxml.html import fromstring
+from searx.utils import extract_text
+from searx.raise_for_httperror import raise_for_httperror
+
+# about
+about = {
+    "website": 'https://www.wordnik.com',
+    "wikidata_id": 'Q8034401',
+    "official_api_documentation": None,
+    "use_official_api": False,
+    "require_api_key": False,
+    "results": 'HTML',
+}
+
+categories = ['general']
+paging = False
+
+word_xpath = '//*[@id="headword"]/text()'
+from_xpath = '//*[@id="define"]/div/h3[1]'
+definitions_xpath = '//*[@id="define"]/div/ul[1]'
+misspelling_xpath = '//*[@id="define"]/div/ul/li/xref'
```

Comment:
Do you have an example (search term) matching this xpath? .. I tried several typos, but I did not find a term where I got a misspelling_xpath in the response from wrodnick.

Reply:
For example `distopia`. It is easier for us - ESLs :-)

Reply:
Ah, Ok .. thanks .. I think it is OK to display it "as is" ..

![image](https://user-images.githubusercontent.com/554536/113854105-05007b80-978e-11eb-9843-90387f1e4883.png)